lib/recognizer_web/controllers/accounts/prompt/verification_controller.ex

```diff
@@ -23,8 +23,8 @@ defmodule RecognizerWeb.Accounts.Prompt.VerificationController do
        ]
        when action in [:resend]
 
-  def new(%{assigns: %{user: %{verified_at: nil}}} = conn, _params) do
-    render(conn, "new.html", resend?: false)
+  def new(%{assigns: %{user: %{verified_at: nil} = user}} = conn, _params) do
+    render(conn, "new.html", resend?: false, email: user.email)
```

Comment:
nitpick, but this could be grabbed from the struct directly as `%{verified_at: nil, email: email}` instead of as a lookup

Reply:
I wrote it this way to be consistent with `resend` - which needs access to the user.